onnxruntime/test/onnx/TestResultStat.h

```diff
@@ -33,12 +33,12 @@ class TestResultStat {
     failed_kernels.insert(s);
   }
 
-  void AddFailedTest(const std::string& s) {
+  void AddFailedTest(const std::pair<std::string,size_t>& p) {
     std::lock_guard<onnxruntime::OrtMutex> l(m_);
-    failed_test_cases.insert(s);
+    failed_test_cases.push_back(p);
   }
 
-  std::unordered_set<std::string> GetFailedTest() {
+  const std::vector< std::pair<std::string, int64_t> >& GetFailedTest() {
```

Comment:
>st() { [](start = 67, length = 6)

const method, now returns a ref to a member

Reply:
It is not a const function. 
And it returns const ref.

Reply:
My comment means it should be a const method

---
In reply to: [287118001](https://github.com/microsoft/onnxruntime/pull/1073#discussion_r287118001) [](ancestors = 287118001)